docs/scuttling.md

```diff
@@ -0,0 +1,74 @@
+## Intro
+
+Scuttling is a LavaMoat optional security feature, first introduced in [#360](https://github.com/LavaMoat/LavaMoat/pull/360).
+
+## Warning ⚠️
+
+This feature is experimental. This means that:
+1. Getting it to work on your LavaMoat protected application without breaking parts of it might require some effort, especially if your application loads JavaScript code outside of LavaMoat protection.
+2. There still might be issues with this feature when trying to use it (please inform us if so!).
+3. The API to activate this feature might (and probably will) change before achieving stability.
+
+## Usage
+
+When using LavaMoat (node/lavapack/browserify), it accepts the argument `scuttleGlobalThis`:
+
+```javascript
+scuttleGlobalThis: {
+  enabled: <boolean>,     // default:false
+  exceptions: []<string>, // default: []
+  scuttler: <string>,     // default:''
+}
+```
+
+* `enabled` - whether to enable scuttling security feature or not
+* `exceptions` - if `enabled:true`, provide a list of properties to avoid from scuttling (which means, not to remove those from the global object)
```

Comment:
nit

```suggestion
* `exceptions` - if `enabled:true`, provide a list of properties to avoid scuttling (which means, not to remove those from the global object)
```

(or _avoid being scuttled_)